packages/ui/core-components/src/lib/unsorted/viz/table/_DataTable.svelte

```diff
@@ -479,6 +479,7 @@
 	{/each}
 
 	<div
+		data-testid={isFullPage ? undefined : `DataTable-${data?.id ?? 'no-id'}`}
```

Comment:
Since there's no better way to query for a DataTable (no title or label), I'm okay with using `data-testid` here